e2e-pw/src/oss/poms/modal/group-actions.ts

```diff
@@ -0,0 +1,40 @@
+import { Page, expect } from "src/oss/fixtures";
+import { ModalPom } from ".";
+
+export class ModalGroupActionsPom {
+  readonly page: Page;
+  readonly modal: ModalPom;
+  readonly assert: ModalGroupActionsAsserter;
+
+  constructor(page: Page, modal: ModalPom) {
+    this.page = page;
+    this.modal = modal;
+    this.assert = new ModalGroupActionsAsserter(this);
+  }
+
+  async getGroupPinnedText() {
+    return this.modal.locator
+      .getByTestId("pinned-slice-bar-description")
+      .textContent();
+  }
+
+  getGroupContainer() {
+    return this.modal.locator.getByTestId("group-container");
+  }
+
+  async selectNthItemFromCarousel(index: number) {
```

Comment:
We're adding some overlapping poms. We can merge this first

Reply:
cool